Tests/WalletConnectTests/PairingEngineTests.swift

```diff
@@ -148,17 +148,13 @@ final class PairingEngineTests: XCTestCase {
         XCTAssertEqual(approvedPairing?.topic, topicB, "The approved pairing must settle on topic B.")
         // TODO: Check if expiry time is correct
     }
-    
-//    func testNotifyOnSessionProposal() {
-//        let topic = "1234"
-//        let proposalExpectation = expectation(description: "on session proposal is called after pairing payload")
-////        engine.sequencesStore.create(topic: topic, sequenceState: sequencePendingState)
-//        try? engine.sequencesStore.setSequence(pendingPairing)
-//        let subscriptionPayload = WCRequestSubscriptionPayload(topic: topic, clientSynchJsonRpc: sessionProposal)
-//        engine.onSessionProposal = { (_) in
-//            proposalExpectation.fulfill()
+//
+//    func testNonControllerExtendFails() {
```

Comment:
Are you going to reactivate or remove this test?

Reply:
I've removed it. We can test here when the "new pairing" is more defined